publication-event-handlers/src/main/java/no/unit/nva/publication/events/handlers/tickets/AcceptedPublishingRequestEventHandler.java

```diff
@@ -59,11 +64,7 @@ protected Void processInputPayload(EventReference input,
         var eventBlob = s3Driver.readEvent(input.getUri());
         var latestUpdate = parseInput(eventBlob);
         if (TicketStatus.COMPLETED.equals(latestUpdate.getStatus()) && publicationIsNotPublished(latestUpdate)) {
-            var userInstance = UserInstance.create(latestUpdate.getOwner(), latestUpdate.getCustomerId());
-            var publication = fetchPublication(userInstance, latestUpdate.extractPublicationIdentifier());
-            attempt(() -> resourceService.publishPublication(userInstance, latestUpdate.extractPublicationIdentifier()))
-                .orElse(fail -> logError(fail.getException()));
-            createDoiRequestIfNeeded(publication);
+            publishPublication(latestUpdate);
```

Comment:
For workflow #1, don't you publish the publication on ticket creation? If that is the case, you should not repeat it here. Or what?

Reply:
We do that, as I can see it it this handler accepts publishingRequests, and it will now publish Publication only if publication is not published

Reply:
This handler for now does following part of requirements:

Workflow #2:

“Unpublished” files are published as part of ticket “approval”/completion.